make-uniform/production/Build Standard Database.Rmd

```diff
@@ -108,19 +107,19 @@ f_vta_survey_path <- paste0(dir_path,
   "VTA/As CSV/VTA_DRAFTFINAL_20171114 NO POUND OR SINGLE QUOTE.csv")
 
 f_output_rdata_path <- paste0(dir_path, 
-  "_data Standardized/survey_standard.RData")
+  "_data Standardized/survey_standard.RDS")
```

Comment:
@jhelsel11:  When using saveRDS, we should use the RDS extension, so users know that it's an `RDS` file and not an `Rdata` file.